### PR TITLE
net-wireless/wavemon: Version bump

### PR DESCRIPTION
**Pull request by alarig**
Package-Manager: Portage-3.0.12, Repoman-3.0.2
Signed-off-by: Alarig Le Lay <alarig@swordarmor.fr

### CODE REVIEW

File: net-wireless/wavemon/files/wavemon-0.9.3-build.patch [hunk 1]
```diff
@@ -0,0 +1,40 @@
+--- a/configure.ac	2020-12-26 17:44:01.616731241 +0100
++++ b/configure.ac	2020-12-26 17:45:02.447292151 +0100
+@@ -22,6 +22,8 @@
```
Comment (juippis) [inline net-wireless/wavemon/files/wavemon-0.9.3-build.patch:3]: We like to have the headers from the original source if you found this somewhere, and if not, still in git-patch format.

File: net-wireless/wavemon/files/wavemon-0.9.3-build.patch [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ # linux/if.h needs 'struct sockaddr'
+ # linux/wireless.h in turn depends on linux/if.h
+--- a/Makefile.in	2017-07-14 21:35:18.154423280 +0300
++++ b/Makefile.in	2017-07-14 21:35:34.613402738 +0300
+@@ -9,7 +9,7 @@
+ install-suid-root: exec_perms = 4755
+ 
+ CC	 ?= @CC@
+-CFLAGS	 ?= @CFLAGS@ @LIBNL3_CFLAGS@
++CFLAGS	 = @CFLAGS@ @LIBNL3_CFLAGS@
```
Comment (juippis) [inline net-wireless/wavemon/files/wavemon-0.9.3-build.patch:27]: However looks like this patch isn't working as you wish,
```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/bin/wavemon
```

File: net-wireless/wavemon/files/wavemon-0.9.3-build.patch (hunk 1 above)
Comment (alarig) [reply net-wireless/wavemon/files/wavemon-0.9.3-build.patch]: I re-created the patch to adapt it to this new version, so I guess that the headers reflect the original source?

File: net-wireless/wavemon/files/wavemon-0.9.3-build.patch (hunk 2 above)
Comment (alarig) [reply net-wireless/wavemon/files/wavemon-0.9.3-build.patch:36]: I removed the CFLAGS from `configure.ac`, so now the CFLAGS are respected.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**